examples/Multibus_simple/Multibus_simple.ino

```diff
@@ -0,0 +1,48 @@
+#include <OneWire.h>
+#include <DallasTemperature.h>
+
+int oneWirePins[]={3,7};//OneWire DS18x20 temperature sensors on these wires
+const int oneWirePinsCount=sizeof(oneWirePins)/sizeof(int);
+
+OneWire ds18x20[oneWirePinsCount];
```

Comment:
This line gives me this error:

```
Multibus_simple:8: error: no matching function for call to 'OneWire::OneWire()'
 OneWire ds18x20[oneWirePinsCount];
                                 ^
```

I guess because there is no constructor that takes an array as argument?